README.md

```diff
@@ -276,6 +276,10 @@ The following boolean configuration options are available:
   hardware, which may become drastically lower in the future. Whether or not
   this feature is enabled, the metadata is all contained within an isolated
   memory region with high entropy random guard regions around it.
+* `CONFIG_BLOCK_OPS_CHECK_SIZE`: `true` or `false` (default) to ensure length
+  parameter of the memcpy/memmove/memset block operations are within
+  approximate bounds to minimize buffer overflows. Note, memset override is
+  currently disabled due to improper behavior.
```

Comment:
Improper behaviour from whom/what?

Reply:
everything hangs

I thought it may be because hm uses memset itself, so I replaced all of its calls to the h_memset_real function but that still had the same behavior

Reply:
it appears that it starts recursing into itself

Reply:
Please add this in the README then :)